### DISCUSSION TITLE
Longer/worst route in OsmAnd

### DISCUSSION THREAD

**Discussion by pablorodz** [osmandapp/OsmAnd#13834] [Maps Related]
Hi,
I was driving through Cordoba, Argentina, and I had an issue with OsmAnd routing which does not happen with [Graphopper ](https://www.openstreetmap.org/directions?engine=graphhopper_car&route=-31.42939%2C-64.18624%3B-31.42401%2C-64.18424#map=17/-31.42632/-64.18382)

![2022-03-02 03_23_49-Note_ 3060657 _ OpenStreetMap — Mozilla Firefox](https://user-images.githubusercontent.com/666466/156289158-26f38a24-4381-4617-85ac-234f6fd8c1fb.png)

The following is the same route in OsmAnd, which made me take Arenal, the roundabout and the tunnel, instead of the shorter and faster route shown above.

![photo4969892665101495008](https://user-images.githubusercontent.com/666466/156288839-9f20a5bd-6cbc-4446-a0dd-db64019380f7.jpg)

**Comment by vshcherb** [comment]
By definition it has higher cost mostly because of traffic signals 200 (https://osmand.net/map?start=-31.429137%2C-64.184850&end=-31.424179%2C-64.184170&mode=car#17/-31.42646/-64.18175) vs 220.
